string pc_name
string time
float64 cpu_per
int64 used_memory
float64 memory_per
string from_node
float64 timestamp